readme.md

```diff
@@ -0,0 +1,89 @@
+This is a docker container intended to act as a build and run environment for
+FreeCAD.
+
+The directories containing FreeCAD's source code and build are not included
+inside the docker image. Instead, they are attached to the docker container
+when you run the container. This allows the built code to have continuity
+across different docker containers, reducing the time for a build to occur, and
+allowing you to use your own editor/IDE outside of the container.
+
+# Image use
+
+## Pull image
+
+```
+docker pull registry.gitlab.com/daviddaish/freecad_docker_env:latest
```

Comment:
Should the `readme.md` in the target `FreeCAD/Docker`-repo contain references to gitlab / daviddaish?
Did [this post](https://www.forum.freecadweb.org/viewtopic.php?p=379912#p379912) get any follow-up?